src/components/document/sort-work-view.tsx

```diff
@@ -15,25 +16,40 @@ import "../thumbnail/document-type-collection.scss";
  * Various options for sorting the display are available - by user, by group, by tools used, etc.
  */
 export const SortWorkView: React.FC = observer(function SortWorkView() {
-  const { appConfig, persistentUI, sortedDocuments } = useStores();
-
-  //*************************** Determine Sort Options & State  ***********************************
-  const {tagPrompt} = appConfig;
+  const { appConfig, investigation, persistentUI, problem, sortedDocuments, unit } = useStores();
+  const { tagPrompt } = appConfig;
+  const { docFilter: persistentUIDocFilter } = persistentUI;
   const sortTagPrompt = tagPrompt || ""; //first dropdown choice for comment tags
   const sortOptions = ["Group", "Name", sortTagPrompt, "Bookmarked", "Tools"];
+  const filterOptions: DocFilterType[] = ["Problem", "Investigation", "Unit", "All"];
   const [sortBy, setSortBy] = useState("Group");
+  const [docFilter, setDocFilter] = useState(persistentUIDocFilter);
```

Comment:
You probably don't need this react state. Just by calling `persistentUI.setDocFilter(filter)` it should trigger a re-render.

Reply:
You're right, it's not needed. Thanks. Removed in latest commit.